docs/docsite/rst/user_guide/playbooks_strategies.rst

```diff
@@ -3,37 +3,31 @@
 Strategies
 ===========
 
-Strategies are a way to control play execution. By default, plays run with a  ``linear`` strategy, in which all hosts will run each task before any host starts the next task, using the number of forks (default 5) to parallelize.
+Strategies control play execution. By default, plays run with the :ref:`linear strategy<linear_strategy>`, in which all hosts will run each task before any host starts the next task, using the number of forks (default 5) to parallelize. To 'batch' the hosts while using the ``linear`` strategy, set the ``serial`` directive to the number of hosts you want to manage in each batch. Ansible will then
```

Comment:
`serial` is not unique to the linear strategy. Please link to https://docs.ansible.com/ansible/latest/reference_appendices/playbooks_keywords.html#playbook-keywords for definitions of play level parameters.

Reply:
Thanks @samdoran. Does 441e69d0b7ca82eeaac239cea2a2ff7afa356a1a address this?